src/main/java/org/spongepowered/api/service/pagination/PaginationList.java

```diff
@@ -103,8 +103,18 @@ static Builder builder() {
      * Send the constructed pagination list to the given source.
      *
      * @param source The source to send to
+     * @param page The page to send
      */
-    void sendTo(MessageReceiver source);
+    void sendTo(MessageReceiver source, int page);
+
+    /**
+     * Send the constructed pagination list to the given source.
```

Comment:
Maybe we should state explicitly that it sends the first page. Just to avoid confusion with a random `current` page.